components/bpmn/bpmn-explorer-web/src/web/js/actions.js

```diff
@@ -455,4 +455,46 @@ function setDatePicker (dateElement) {
     });
 }
 
+//Process Instance Count
+function selectProcessForInstanceCount(){
+    var x = document.getElementById("processInstanceCount").value;
+    var url = httpUrl + "/" + CONTEXT + "/stats?update=true&option=processinstancecount&id=" + x ;
+
+    $.ajax({
+        type: 'GET',
+        contentType: "application/json",
+        url: url,
+        success: function (data) {
+
+            var array = eval('('+data+')');
+            google.load("visualization", "1", {packages:["corechart"]});
+            google.setOnLoadCallback(drawChart(array));
+
+
+            function drawChart(data) {
+                var dataArr = [['Process Name', 'Instance Count']];
+                for(var i = 0;i < data.length;i++){
```

Comment:
Already fixed and merged
https://github.com/wso2/carbon-business-process/blob/master/components/bpmn/bpmn-explorer-web/src/web/js/actions.js#L656